ruby/basic_ruby/arrays.md

```diff
@@ -139,14 +139,14 @@ Here is a brief look at some other common array methods you might run into:
 
 1. Read Launch School's chapter on [Arrays](https://launchschool.com/books/ruby/read/arrays), following along with the exercises using irb or any other REPL, such as [replit.com](https://replit.com/languages/ruby).
 2. Read [The Definitive Guide to Ruby Arrays](https://www.rubyguides.com/2015/05/ruby-arrays/) by Jesus Castello.
-3. Complete the [array](https://github.com/TheOdinProject/ruby-exercises/tree/main/ruby_basics) exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises) that you previously cloned. 
-4. Let's get some practice reading documentation. You'll feel just like a real programmer! What do you think the methods `#clear`, `#insert`, `#sample`, `#shuffle`, and `#uniq` do? Look these methods up in the [Array class documentation](https://ruby-doc.org/core/Array.html). Were you close?
+3. Complete the [array](https://github.com/TheOdinProject/ruby-exercises/tree/main/ruby_basics) exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises) that you previously cloned.
+4. Let's get some practice reading documentation. You'll feel just like a real programmer! What do you think the methods `#clear`, `#insert`, `#sample`, `#shuffle`, and `#uniq` do? Look these methods up in the [Array class documentation](https://docs.ruby-lang.org/en/3.2/Array.html). Were you close?
 </div>
 
 ### Additional resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-* Look over the latest [Ruby API documentation](http://ruby-doc.org/) on arrays by clicking on the "Current" tab in the top left corner, scrolling down to the "Class and Module Index" section on the left sidebar, and clicking on "Array". There, you'll find the most up-to-date documentation on the various methods available to the Array object along with explanations and examples.
+* Look over the latest [Ruby API documentation](https://docs.ruby-lang.org/) on arrays by choosing a language and Ruby version (e.g. English, Ruby 3.2), then scrolling down to the "Class and Module Index" section on the left sidebar and clicking on "Array". There, you'll find the most up-to-date documentation on the various methods available to the Array object along with explanations and examples.
```

Comment:
What do you think about just linking to https://docs.ruby-lang.org/en/3.2/Array.html here?

Reply:
I think it's a great idea to at least eliminate this second explanation on line 149, replacing it with a simple link.

One could also argue for a similar change to the first explanation on line 111. I think I would agree with that, but I'm not attached.

Reply:
Oh, yes! Making a change to line 111 would be useful too. 👍